chat.tox.qTox.json

```diff
@@ -0,0 +1,44 @@
+{
+	"app-id": "chat.tox.qTox",
+	"runtime": "org.kde.Platform",
+	"sdk": "org.kde.Sdk",
+	"runtime-version": "5.10",
+	"command": "qtox",
+	"finish-args": [
+		"--share=network",
+		"--socket=pulseaudio",
+		"--socket=wayland",
+		"--socket=x11",
+		"--filesystem=host",
+		"--device=all"
+	],
+	"build-options": {
+		"cflags": "-O3 -DSQLITE_HAS_CODEC",
+		"cxxflags": "-O3"
+	},
+	"modules": [
+		{
+			"name": "sqlcipher",
+			"config-opts": ["--enable-tempstore=yes", "--disable-tcl"],
+			"sources": [{"type": "git", "url": "https://github.com/sqlcipher/sqlcipher", "tag": "v3.4.2", "commit": "c6f709fca81c910ba133aaf6330c28e01ccfe5f8"}]
+		},
+		{
+			"name": "libsodium",
```

Comment:
These will install headers, pkg-config files, etc. See the build directory for extraneous files and use the `cleanup` property.